package.json

```diff
@@ -20,51 +20,51 @@
     "build-docs": "node_modules/.bin/typedoc --tsconfig tsconfig.json"
   },
   "dependencies": {
-    "@nypl/dgx-svg-icons": "^0.3.9",
+    "@nypl/dgx-svg-icons": "^0.3.13",
```

Comment:
Are all these updates from this PR or from a different branch that just merged into this one?

Reply:
Just from pulling the newest version of `master`.